libshaderc/include/shaderc.h

```diff
@@ -121,6 +127,11 @@ void shaderc_compile_options_add_macro_definition(
 void shaderc_compile_options_set_disassembly_mode(
     shaderc_compile_options_t options);
 
+// Sets the compiler mode to force (without any verification) the default
+// version and profile for compilation.
```

Comment:
I'd like this to say that it's forcing the *GLSL* language version and profile.  Should also say that the version number is the same as would appear in the #version annotation in the source. 

The blurb should describe how this setting interacts with the #version annotation in the source, if any such annotation exists.
(You have it in your test cases)